### PR TITLE
update: add 1.5.31 release details

### PR DESCRIPTION
**Pull request by shadrina**

### CODE REVIEW

**Comment by shadrina** [comment]
- Will trigger staging tomorrow in the morning
- The link to `kotlinx-nodejs` version is questionable.

File: docs/topics/releases.md [hunk 1]
```diff
@@ -49,6 +49,27 @@ You can also use [preview versions of Kotlin](eap.md#build-details).
         <th>Build highlights</th>
         <th>Recommended kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.5.31</strong>
+            <p>Released: <strong>September 14, 2021</strong></p>
```
Comment (koshachy) [inline docs/topics/releases.md:54]: Don't forget to change the date

File: docs/v.list [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
-    <var name="kotlinVersion" value="1.5.30" type="string"/>
+    <var name="kotlinVersion" value="1.5.31" type="string"/>
 
     <var name="kotlinEapVersion" value="1.5.30-RC" type="string"/>
 
-    <var name="kotlinLatestUrl" value="https://github.com/JetBrains/kotlin/releases/tag/v1.5.30" type="string"/>
+    <var name="kotlinLatestUrl" value="https://github.com/JetBrains/kotlin/releases/tag/v1.5.31" type="string"/>
 
-    <var name="kotlinReleaseDate" value="August 24, 2021" type="string"/>
+    <var name="kotlinReleaseDate" value="September 14, 2021" type="string"/>
```
Comment (koshachy) [inline docs/v.list:13]: Same here

**Review by koshachy — APPROVED** [review]

File: docs/topics/releases.md (hunk 1 above)
Comment (shadrina) [reply docs/topics/releases.md]: Thanks, fixed

File: docs/v.list (hunk 2 above)
Comment (shadrina) [reply docs/v.list]: Thanks, fixed

**Review by woainikk — APPROVED** [review]

**Pull request merged**